resources/migrations/000_migrations.yaml

```diff
@@ -11637,6 +11637,157 @@ databaseChangeLog:
             columnName: creator_id
             columnDataType: int
 
+  - changeSet:
```

Comment:
I'd add a comment saying the migration on top of this saying from `v44.00-002` to `v44.00-011` is adding entitie_id to a list of tables.

It'll make it easier to browse the migration IMO.